src/azure-cli/azure/cli/command_modules/signalr/tests/latest/test_signalr_commands.py

```diff
@@ -30,17 +32,19 @@ def test_signalr_commands(self, resource_group):
             'sku': sku,
             'unit_count': unit_count,
             'tags': '{}={}'.format(tags_key, tags_val),
+            'enable_message_logs': enable_message_logs,
             'service_mode': service_mode,
             'allowed_origins': ' '.join(allowed_origins),
             'updated_sku': updated_sku,
             'updated_tags': '{}={}'.format(tags_key, updated_tags_val),
             'update_service_mode': update_service_mode,
+            'update_enable_message_logs': update_enable_message_logs,
             'added_allowed_origins': ' '.join(added_allowed_origins),
             'default_action': "Deny",
         })
 
         # Test create
-        self.cmd('az signalr create -n {signalr_name} -g {rg} --sku {sku} --unit-count {unit_count} -l {location} --tags {tags} --service-mode {service_mode} --allowed-origins {allowed_origins} --default-action {default_action}',
+        self.cmd('az signalr create -n {signalr_name} -g {rg} --sku {sku} --unit-count {unit_count} -l {location} --tags {tags} --service-mode {service_mode} --enable-message-logs {enable_message_logs} --allowed-origins {allowed_origins} --default-action {default_action}',
```

Comment:
please add test for `--enable-message-logs false` in create or update command. 
With such test you will find your enable_message_logs will always be true because if you use string